src/client/routes/lobby/GameList.tsx

```diff
@@ -50,4 +44,23 @@ class GameList extends React.Component<any, any> {
   }
 }
 
+interface IGameListProps extends RouteComponentProps<any>, WithStyles<typeof styles> {
+  title: string,
+  games: global.IGameRef[],
+  openDialog: any,
```

Comment:
openDialog = any, or function?

Reply:
Having `openDialog: function` Gives `[ts] Cannot find name 'function'.`.

Reply:
Assuming there's no return value: 
openDialog: () => void

Of course it would be better if we can avoid passing and binding functions, but I guess that's v2.0 with redux? :)

Reply:
Fixed. Even if we use redux we probably cant avoid passing down functions. It would make it a lot cleaner in other ways though.

Reply:
Then we should look into some other react compliant event publishing system. Passing and binding these is ugly.